codeclimate-radon

```diff
@@ -22,6 +32,5 @@ if os.path.exists("/config.json"):
                 python_paths.append(i)
         include_paths = python_paths
 
-if len(include_paths) > 0:
-    args = " ".join(include_paths)
-    os.system("radon cc {0} -n{1} --codeclimate".format(args, threshold))
+args = " ".join(include_paths)
```

Comment:
I know this isn't new, but is this safe for paths with spaces in them? I don't see anywhere we do sanitizing. If not, maybe fix it here or file a small bug card?